iceoryx_posh/test/moduletests/test_popo_subscriber_port.cpp

```diff
@@ -301,3 +301,165 @@ TEST_F(SubscriberPort_test, nackWhenNotWaitingForResultsInError)
     EXPECT_FALSE(maybeCaproMessage.has_value());
     EXPECT_TRUE(errorHandlerCalled);
 }
+
+class SubscriberPortMultiProducer_test : public Test
```

Comment:
Please move the following code into a separate file called `test_popo_subscriber_port_multi_producer.cpp`

Reply:
This is a question about SW units. We have a separation of data structures in the shared memory and access classes without own data members (besides pointer to this shared memory object). We also separate the multi-threaded ones in access classes for different threads of execution (like RouDi and User). You could have everything in one class, it is more an architectural decision to do it that way. For me all these classes are the SubscriberPort SW unit. And if each unit has a unit test this would all be in one file.

Reply:
Can we write this down somewhere?